app/src/main/java/com/ggetters/app/ui/central/sheets/AccountEditBottomSheet.kt

```
@@ -159,40 +161,120 @@ class AccountEditBottomSheet : BottomSheetDialogFragment() {
             }
             
-            if (updatedUser.email.isNullOrBlank()) {
-                etEmail.error = "Email is required"
-                return
+            lifecycleScope.launch {
+                try {
+                    profileViewModel.updateUserProfile(updatedUser)
+                    onUserUpdated?.invoke(updatedUser)
+                    Snackbar.make(requireView(), "Profile updated successfully", Snackbar.LENGTH_SHORT).show()
+                    dismiss()
+                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)